Gemfile

```diff
@@ -43,7 +43,7 @@ gem "jquery-ui-rails"
 gem "simple_form", '~> 3.1.0'
 gem 'nokogiri'
 gem 'equivalent-xml'
-gem 'solr_wrapper'
+gem 'solr_wrapper', '0.7.3'
```

Comment:
And, you can configure solr_wrapper to use old versions of solr. Pinning the `solr_wrapper` dependency is probably not the right way to fix this. A better way would be creating a `.solr_wrapper` configuration file with, e.g.:

```yaml
version: '5.5.1'
```